faq-bundle/src/Resources/contao/modules/ModuleFaq.php

```diff
@@ -103,6 +103,11 @@ public function getSearchablePages($arrPages, $intRoot=0, $blnIsSitemap=false)
 				{
 					while ($objItems->next())
 					{
+						if ($blnIsSitemap && $objItems->robots === 'noindex,nofollow')
```

Comment:
should this check the page settings as well?

Reply:
It's already checked above, so no need to check it again https://github.com/contao/contao/blob/449a6fbb3717af62b5f40f1f71cd9f5c71fb87f1/faq-bundle/src/Resources/contao/modules/ModuleFaq.php#L87